src/Client/Grpc/GrpcDurableTaskClient.cs

```diff
@@ -320,13 +329,13 @@ OrchestrationMetadata CreateMetadata(P.OrchestrationState state, bool includeInp
 
     AsyncDisposable BuildChannel(out Channel channel)
     {
-        if (this.grpcOptions.Channel is Channel c)
+        if (this.options.Channel is Channel c)
         {
             channel = c;
             return default;
         }
 
-        string address = string.IsNullOrEmpty(this.grpcOptions.Address) ? "127.0.0.1:4001" : this.grpcOptions.Address!;
+        string address = string.IsNullOrEmpty(this.options.Address) ? "127.0.0.1:4001" : this.options.Address!;
```

Comment:
BTW, I've been meaning to open a PR for this, but would you be willing to change this default address to `"localhost:4001"` for me? I noticed that `localhost` works better in mixed environments, such as when a client is running on Windows and the server is running on a local WSL2 virtual machine (I discovered this when testing this SDK against Dapr). `127.0.0.1` from a Windows client unfortunately wasn't able to reach an endpoint running on WSL2.

Reply:
Done